### PR TITLE
Fix some bugs

### PR DESCRIPTION
**Pull request by ImenBizid**
List of bugs
1. Machine Leaning Bucket
     1.1  Fill_NanNs task: has python Language instead cpyton Language.
2. Machine Learning Workflows Bucket
     2.1 Diabetics_Detection_Using_K_Means workflow: the Download_Model task doesn't  download model correctly (zero bytes).
     2.2 California_Housing_Prediction_Using_TPOT_Regressor: it takes a long time to run.

### CODE REVIEW

File: MLBasics/resources/catalog/California_Housing_Prediction_Using_TPOT_Regressor.xml [hunk 1]
```diff
@@ -74,11 +76,13 @@ import pandas as pd
 import numpy as np
 
 from sklearn.datasets import fetch_california_housing
+from sklearn.model_selection import train_test_split
 california_housing = fetch_california_housing()
 dataframe_load = pd.DataFrame(california_housing.data)
 dataframe_load.columns = california_housing.feature_names 
 data_label = california_housing.target
 dataframe = dataframe_load.assign(LABEL=data_label)
+dataframe, neglected = train_test_split(dataframe, test_size=0.98)
```
Comment (carolinepacheco) [inline MLBasics/resources/catalog/California_Housing_Prediction_Using_TPOT_Regressor.xml]: Why a train_test_split in the Load_Californa_Housing_Dataset task?  not clear...

File: MLBasics/resources/catalog/Diabetics_Detection_Using_K_Means.xml [hunk 2]
```diff
@@ -1,27 +1,29 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (carolinepacheco) [inline MLBasics/resources/catalog/Diabetics_Detection_Using_K_Means.xml]: Whats the difference from this workflow and the previous one?

File: MLBasics/resources/catalog/California_Housing_Prediction_Using_TPOT_Regressor.xml (hunk 1 above)
Comment (ImenBizid) [reply MLBasics/resources/catalog/California_Housing_Prediction_Using_TPOT_Regressor.xml]: it is used to reduce the dataset size as the full dataset is huge.

File: MLBasics/resources/catalog/Diabetics_Detection_Using_K_Means.xml (hunk 2 above)
Comment (ImenBizid) [reply MLBasics/resources/catalog/Diabetics_Detection_Using_K_Means.xml]: just corrected the the link between train model and download model.

**Pull request merged**